.travis.yml

```diff
@@ -1,11 +1,11 @@
 sudo: false
 language: go
 go:
-  - 1.5.x
```

Comment:
Dropping 1.5 since it fails with:

```
...

gopkg.in/ini.v1
# gopkg.in/ini.v1
../../../gopkg.in/ini.v1/file.go:30: undefined: LoadOptions
../../../gopkg.in/ini.v1/file.go:31: undefined: dataSource
The command "eval go get -t -v ./... " failed 3 times.
The command "go get -t -v ./..." failed and exited with 2 during .
```